tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -142,77 +144,114 @@ static std::string removeExt(const std::string &FileName) {
 
 static ExitOnError ExitOnErr;
 
-static int convertLLVMToSPIRV(const SPIRV::TranslatorOpts &Opts) {
-  LLVMContext Context;
+static void error(const Twine &Msg) {
+  errs() << "llvm-spirv: " << Msg << '\n';
+  exit(1);
+}
+
+static void error(std::error_code EC, const Twine &Prefix) {
+  if (EC)
+    error(Prefix + ": " + EC.message());
+}
+
+static void saveListToFile(std::string Filename, std::string Content) {
```

Comment:
What about passing both arguments as const reference instead of making copies?

Reply:
Looks good, done.